host-vm1.yml

```diff
@@ -26,5 +26,12 @@
     ip_address:
     - 'lagg0|140.211.10.238'
     - 'vlan0|192.168.80.238'
+  - role: jail
+    name: bslave1
+    # XXX: this is interesting and needs to be documented
```

Comment:
Domain names: what domains we want to use (looks like `buildbot.net` and `int.buildbot.net`; when we use only former, when we use only latter and when we use both.